core/state/journal.go

```diff
@@ -127,9 +127,7 @@ type (
 		hash common.Hash
 	}
 	touchChange struct {
-		account   *common.Address
-		prev      bool
```

Comment:
What? We can remove those?

Reply:
Oh, right, I changed that

https://github.com/ethereum/go-ethereum/commit/958ed4f3d977b08465915e475e11aaab3d2dc574